### DISCUSSION TITLE
Live Filter like Snapchat using OpenCV for Overlays

### DISCUSSION THREAD

**Discussion by tools9112** [photobooth-app/photobooth-app#333] [Ideas]
Hi,

I saw a feature in a different release of another photobooth repository [Pi-Photobooth by mdegrazia](https://github.com/mdegrazia/Pi-Photobooth). They created overlays, like a mustache filter similar to Snapchat, which they called "Live Filters."

Would it be possible to add this feature?

**Comment by mgineer85** [comment]
While it would be cool, I think it’s quite effort to implement right. 
 I do not have researched possible ways to implement with up to date filter algorithms, not giving you some outdated 90ies experience 🙃

I leave it open here for discussion, maybe somebody wants to try it at one point.

**Reply by tools9112** [reply]
This is the original blog post where I saw that repository: https://anotherpiblog.blogspot.com/2017/06/raspberry-pi-photobooth-with-bluetooth.html. OpenCV is quite prevalent and should provide a good modern experience, I think. :)

**Reply by mgineer85** [reply]
Yes, but there are many algorithms , some more elaborated, some with better performance and some limited to specific platforms.
Unjust found a more recent description https://learnopencv.com/create-snapchat-instagram-filters-using-mediapipe/
